homeassistant/components/daikin/sensor.py

```diff
@@ -83,30 +89,45 @@ class DaikinSensorEntityDescription(SensorEntityDescription, DaikinRequiredKeysM
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=PERCENTAGE,
         value_func=lambda device: device.humidity,
+        outdoor_sensor=False,
     ),
     DaikinSensorEntityDescription(
-        key=ATTR_TOTAL_POWER,
+        key=ATTR_ALL_POWER,
         name="Estimated Power Consumption",
         device_class=SensorDeviceClass.POWER,
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=POWER_KILO_WATT,
         value_func=lambda device: round(device.current_total_power_consumption, 2),
+        outdoor_sensor=True,
     ),
     DaikinSensorEntityDescription(
         key=ATTR_COOL_ENERGY,
         name="Cool Energy Consumption",
         icon="mdi:snowflake",
         device_class=SensorDeviceClass.ENERGY,
         native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        entity_registry_enabled_default=False,
         value_func=lambda device: round(device.last_hour_cool_energy_consumption, 2),
+        outdoor_sensor=False,
     ),
     DaikinSensorEntityDescription(
         key=ATTR_HEAT_ENERGY,
         name="Heat Energy Consumption",
         icon="mdi:fire",
         device_class=SensorDeviceClass.ENERGY,
         native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        entity_registry_enabled_default=False,
         value_func=lambda device: round(device.last_hour_heat_energy_consumption, 2),
+        outdoor_sensor=False,
+    ),
+    DaikinSensorEntityDescription(
+        key=ATTR_ENERGY_TODAY,
+        name="Energy Consumption",
```

Comment:
Note our guidelines for lowercase / uppercase letters in entity names.

Reply:
Should be okay now